### PR TITLE
testcase: fix ci for datasource/alicloud_cs_kubernetes_clusters and resource/alicloud_cs_kubernetes

### PR DESCRIPTION
**Pull request by Pangjiping**

### CODE REVIEW

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSManagedKubernetesClustersDataSource -timeout=40m -cover -coverprofile CS-2022-05-11.out
=== RUN   TestAccAlicloudCSManagedKubernetesClustersDataSource
--- PASS: TestAccAlicloudCSManagedKubernetesClustersDataSource (871.45s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesAddonMetadataDataSource -timeout=40m
=== RUN   TestAccAlicloudCSKubernetesAddonMetadataDataSource
--- PASS: TestAccAlicloudCSKubernetesAddonMetadataDataSource (889.99s)
PASS

**Comment by Pangjiping** [comment]
% TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesAddonsDataSource -timeout=40m
=== RUN   TestAccAlicloudCSKubernetesAddonsDataSource
--- PASS: TestAccAlicloudCSKubernetesAddonsDataSource (894.75s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesPermissionDataSource -timeout=40m
=== RUN   TestAccAlicloudCSKubernetesPermissionDataSource
--- PASS: TestAccAlicloudCSKubernetesPermissionDataSource (900.04s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesClustersDataSource -timeout=40m
=== RUN TestAccAlicloudCSKubernetesClustersDataSource
--- PASS: TestAccAlicloudCSKubernetesClustersDataSource (931.21s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSEdgeKubernetesClustersDataSource -timeout=40m
=== RUN   TestAccAlicloudCSEdgeKubernetesClustersDataSource
--- PASS: TestAccAlicloudCSEdgeKubernetesClustersDataSource (1082.28s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesPermissions_basic -timeout=40m
=== RUN   TestAccAlicloudCSKubernetesPermissions_basic
--- PASS: TestAccAlicloudCSKubernetesPermissions_basic (807.73s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesNodePool_basic -timeout=40m
=== RUN   TestAccAlicloudCSKubernetesNodePool_basic
--- PASS: TestAccAlicloudCSKubernetesNodePool_basic (670.27s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesNodePoolWithNodeCount_basic -timeout=40m
=== RUN TestAccAlicloudCSKubernetesNodePoolWithNodeCount_basic
--- PASS: TestAccAlicloudCSKubernetesNodePoolWithNodeCount_basic (692.56s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesNodePool_Spot -timeout=40m
=== RUN TestAccAlicloudCSKubernetesNodePool_Spot
--- PASS: TestAccAlicloudCSKubernetesNodePool_Spot (642.37s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesNodePool_autoScaling -timeout=40m
=== RUN   TestAccAlicloudCSKubernetesNodePool_autoScaling
--- PASS: TestAccAlicloudCSKubernetesNodePool_autoScaling (155.29s)
PASS

**Comment by Pangjiping** [comment]
$ TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudCSKubernetesNodePool_BYOK-timeout=40m
=== RUN TestAccAlicloudCSKubernetesNodePool_BYOK
--- PASS: TestAccAlicloudCSKubernetesNodePool_BYOK (720.55s)
PASS

File: alicloud/resource_alicloud_cs_kubernetes.go [hunk 1]
```diff
@@ -183,6 +183,7 @@ func resourceAlicloudCSKubernetes() *schema.Resource {
 			"worker_disk_size": {
 				Type:         schema.TypeInt,
 				Optional:     true,
+				Default:      40,
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_cs_kubernetes.go:186]: 这个应该是 Computed

File: alicloud/resource_alicloud_cs_managed_kubernetes.go [hunk 2]
```diff
@@ -76,6 +76,7 @@ func resourceAlicloudCSManagedKubernetes() *schema.Resource {
 			"worker_disk_size": {
 				Type:         schema.TypeInt,
 				Optional:     true,
+				Default:      40,
```
Comment (xiaozhu36) [inline alicloud/resource_alicloud_cs_managed_kubernetes.go:79]: 还有这个

**Review by xiaozhu36 — CHANGES_REQUESTED** [review]

**Pull request merged**